src/aws/osml/model_runner/database/region_request_table.py

```diff
@@ -152,3 +205,32 @@ def get_region_request(self, region_id: str, image_id: str) -> Optional[RegionRe
         except Exception as err:
             logger.warning(GetRegionRequestItemException(f"Failed to get RegionRequestItem! {err}"))
             return None
+
+    def add_tile(self, image_id: str, region_id: str, tile: ImageRegion, state: str) -> dict[str, Any]:
+        """
+        Append tile to the 'succeeded_tiles' property of a DDB item in the table.
```

Comment:
Comments no longer match the implementation. Not always adding to the "succeeded_tiles" property.

Reply:
Updated comments